index.js

```diff
@@ -36,22 +31,70 @@ const setLazyProperty = (object, property, get) => {
 	});
 };
 
-/** @type {typeof import('color-convert')} */
-let colorConvert;
-const makeDynamicStyles = (wrap, targetSpace, identity, isBackground) => {
-	if (colorConvert === undefined) {
-		colorConvert = require('color-convert');
+// From https://github.com/Qix-/color-convert/blob/3f0e0d4e92e235796ccb17f6e85c72094a651f49/conversions.js
+const colorConversions = {
+	rgb: {
+		ansi256(r, g, b) {
+			// We use the extended greyscale palette here, with the exception of
+			// black and white. normal palette only has 4 greyscale shades.
+			if (r === g && g === b) {
+				if (r < 8) {
+					return 16;
+				}
+
+				if (r > 248) {
+					return 231;
+				}
+
+				return Math.round(((r - 8) / 247) * 24) + 232;
+			}
+
+			return 16 +
+				(36 * Math.round(r / 255 * 5)) +
+				(6 * Math.round(g / 255 * 5)) +
+				Math.round(b / 255 * 5);
+		}
+	},
+	hex: {
+		rgb(hex) {
+			const match = hex.toString(16).match(/[a-f0-9]{6}|[a-f0-9]{3}/i);
```

Comment:
```suggestion
			const match = hex.toString(16).match(/[a-f\d]{6}|[a-f\d]{3}/i);
```

And use named regex groups.

And I prefer using `.exec` instead of `.match`.

Reply:
Named capture groups are only supported in Node.js 10